### PR TITLE
[modular][ready] kilts, a few other things but A SEC KILT

### PR DESCRIPTION
**Pull request by thestubborn**
cowboys are, forgive my pun, old hat, the new craze is highland culture, shockingly not forced at gunpoint by lea

this adds a kilt for security, an Irish tactical outfit since Scotland is just an Irish tribe anyway (Scoti, google it) a blue trenchcoat of a person oft mistook for being from the teddy bear, and a butler tailcoat, for the caretakers among you

### CODE REVIEW

**Comment by thestubborn** [comment]
was this an excuse to flex my rebel song knowledge?
no it was an excuse to get more Scottish shit and reference doctor who

**Comment by thestubborn** [comment]
aaaaaaaaaaaaaaaaaaaaaaaaaaa

**Comment by thestubborn** [comment]
it worked

disband the psni 
trans rights

**Comment by thestubborn** [comment]
i know the sprite's shit


i attended like one art class sue me

**Comment by thestubborn** [comment]
the check cleared, i can stop pretending

File: modular_skyrat/modules/customization/modules/clothing/suits/misc.dm [hunk 1]
```diff
@@ -58,7 +57,7 @@
 	icon_state = "hawaiian_purple"
 	desc = "Strangely en vouge with aviator wearing shibas."
 	body_parts_covered = CHEST|GROIN
-	mutant_variants = NONE
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/clothing/suits/misc.dm:61]: Why are you removing these?

Comment (thestubborn) [reply modular_skyrat/modules/customization/modules/clothing/suits/misc.dm:61]: Oh, I thought another pr done that already and I wanted to copy over the changes, I'll re addit

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**